File: Ticketing App/Ticketing App/Models/IncidentConversation.cs
using Microsoft.AspNetCore.Authorization;
using System;
using System.Collections.Generic;

namespace Ticketing_App.Models
{
    [Authorize]
    public partial class IncidentConversation
    {
        public int Id { get; set; }
        public string commentedBy { get; set; }
        public string comment { get; set; }
        public string Status { get; set; }
        public DateTime? CreatedDate { get; set; }
        public Guid? IncidentId { get; set; }
        public Incident Incident { get; set; }

    }
}

File: Ticketing App/Ticketing App/ClientApp/src/app/Models/incidentconversation.ts
import { Guid } from 'guid-typescript';

export class Incidentconversation {

  ID: number; 
  IncidentCode: string; 
  RaisedBy: string;
  ResolveDescription: string;
  ResponseBy: string;
  ResponseDescription:string 
  Status: string;
  createdDate: Date;
  IncidentId: Guid;
  commentedBy: string;
  comment: string;

}
